### DISCUSSION TITLE
pip install uniswap-python not getting latest files??

### DISCUSSION THREAD

**Discussion by hueit-og** [uniswap-python/uniswap-python#184] [Q&A] [answered]
This could just be me, however I ran into an issue with excessive approvals that I think is related to an outdated package on PyPi.

I found the excessive approvals was fixed in issue #136 around mid July, however when I compared the decorator code on github with what was installed via the 'pip install' I noticed they were different (note - I ran the pip install around 1-oct).

Digging a little deeper it appears the pypi repository is using v0.5.2 from Jun 23, 2021, which I suspect doesn't include any code changes after that date??

I'm just speculating here as I don't really know how the 'pip install' works, but google tells my it uses the pypi repository. :)

**Comment by ErikBjare** [comment, marked as answer]
I just released a new version and updated the package on PyPI, should work fine now.
